repo_config/repo_config.py

```diff
@@ -0,0 +1,87 @@
+class RepoConfig:
+    def __init__(self, config={}, defaults={}):
+        self.config = config
+        self.defaults = defaults
+
+    def get_headers_for_path(self, path_to_match):
+        '''
+        Determine the headers that apply to particular file
+        '''
+
+        # Don't think I need a deep copy here
```

Comment:
Feel like this is a habit from writing JS. 😆

Reply:
I specced a bit out in https://github.com/18F/federalist-garden-build/issues/197 and just pushed up some comments to the code that reflect this. One thing to look it is having an explicit spec in the code itself and validating the config against that during the build and throwing errors. I think that would be a nice thing to add.